test/unified_format.py

```diff
@@ -688,34 +706,11 @@ def _databaseOperation_aggregate(self, target, *args, **kwargs):
     def _collectionOperation_aggregate(self, target, *args, **kwargs):
         return self.__entityOperation_aggregate(target, *args, **kwargs)
 
-    def _collectionOperation_bulkWrite(self, target, *args, **kwargs):
-        self.__raise_if_unsupported('bulkWrite', target, Collection)
-        write_result = target.bulk_write(*args, **kwargs)
-        return parse_bulk_write_result(write_result)
-
     def _collectionOperation_find(self, target, *args, **kwargs):
         self.__raise_if_unsupported('find', target, Collection)
         find_cursor = target.find(*args, **kwargs)
         return list(find_cursor)
 
-    def _collectionOperation_findOneAndReplace(self, target, *args, **kwargs):
-        self.__raise_if_unsupported('findOneAndReplace', target, Collection)
-        return target.find_one_and_replace(*args, **kwargs)
-
-    def _collectionOperation_findOneAndUpdate(self, target, *args, **kwargs):
```

Comment:
Hmm not sure why I had these wrappers in place... They don't seem to do anything. 
I think it might have been just because these ops are explicitly called out in the Unified Test Spec but not sure. Thanks for removing.

Reply:
Yeah I see no reason to keep them either.